src/test/cli/radosgw-admin/help.t

```diff
@@ -369,11 +369,16 @@
      --rgw-obj-fs                  the field separator that will separate the rados object name from the rgw object name;
                                    additionally rados objects for incomplete multipart uploads will not be output
   
+  Bucket list objects options:
+     --max-entries                 max number of entries listed, default 1000, absolute maximum 10000
+     --marker                      the marker used to specify on which entry the listing begins, default none (i.e., very first entry)
+
     --conf/-c FILE    read configuration from the given configuration file
     --id ID           set ID portion of my name
     --name/-n TYPE.ID set name
     --cluster NAME    set cluster name (default: ceph)
     --setuser USER    set uid to user or uid (and gid to user's gid)
     --setgroup GROUP  set gid to group or gid
     --version         show version and quit
-  
+ 
+ 
```

Comment:
the 'make check' failure is complaining about the extra whitespace here and above
```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/cli/radosgw-admin/help.t: failed
--- /home/jenkins-build/build/workspace/ceph-pull-requests/src/test/cli/radosgw-admin/help.t
+++ /home/jenkins-build/build/workspace/ceph-pull-requests/src/test/cli/radosgw-admin/help.t.err
@@ -373,7 +373,7 @@
   Bucket list objects options:
      --max-entries                 max number of entries listed, default 1000, absolute maximum 10000
      --marker                      the marker used to specify on which entry the listing begins, default none (i.e., very first entry)
-
+  
     --conf/-c FILE    read configuration from the given configuration file
     --id ID           set ID portion of my name
     --name/-n TYPE.ID set name
@@ -381,5 +381,7 @@
     --setuser USER    set uid to user or uid (and gid to user's gid)
     --setgroup GROUP  set gid to group or gid
     --version         show version and quit
+  
+
```